src/executors/ReplExecutor.ts

```diff
@@ -93,11 +94,12 @@ export default abstract class ReplExecutor extends AsyncExecutor {
     }
     
     stop(): Promise<void> {
-        return new Promise((resolve, reject) => {
+        return new Promise(async (resolve, reject) => {

```

Comment:
Is it possible to avoid an async callback (see https://stackoverflow.com/a/43050114)? Could the `killWithChildren` method be made synchronous instead?

Reply:
Sure! That sounds like a good idea; thank you for the SO post. That clarifies really well. Thanks!